protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectschema/EnumSchemaTypeConverter.java

```diff
@@ -0,0 +1,39 @@
+package com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema;
+
+import com.google.protobuf.DescriptorProtos;
+import org.apache.kafka.connect.data.Schema;
+import java.util.Map;
+
+import static com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema.ProtobufSchemaConverterConstants.PROTOBUF_ENUM_NAME;
+import static com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema.ProtobufSchemaConverterConstants.PROTOBUF_ENUM_VALUE;
+
+public class EnumSchemaTypeConverter implements SchemaTypeConverter {
+
+    public DescriptorProtos.FieldDescriptorProto.Builder toProtobufSchema(Schema schema, DescriptorProtos.DescriptorProto.Builder descriptorProto, DescriptorProtos.FileDescriptorProto.Builder fileDescriptorProtoBuilder) {
```

Comment:
Same as Ravi's comment: this line contains too many characters. By default the checkstyle should require 80 characters per line. It is also what Intellij enforces.